.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Sming/gdbstub"]
```

Comment:
Well, using submodules has quite some advantages. One of them is that we can easily sync and use the latest version of GDBStub without much effort.